.github/workflows/update-make-docs.yml

```
@@ -3,10 +3,18 @@ on:
   schedule:
     - cron: '0 7 * * 1-5'
   workflow_dispatch:
+
+permissions:
+  contents: read
+
 jobs:
   main:
     if: github.repository == 'grafana/alloy'
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v4
-      - uses: grafana/writers-toolkit/update-make-docs@update-make-docs/v1
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          persist-credentials: false
+      - uses: grafana/writers-toolkit/update-make-docs@f65819d6a412b752c0e0263375215f049507b0e6 # update-make-docs/v1 # zizmor: ignore[unpinned-uses]
+        # This internal action's version is left as a tag instead of a pinned hash because renovate
+        # does not support this type of tag/version format without custom configuration.
```

```suggestion
      - uses: grafana/writers-toolkit/update-make-docs@f65819d6a412b752c0e0263375215f049507b0e6 # update-make-docs/v1 
```

This comment seems outdated, possibly it's outdated on the `main` branch too so feel free to ignore this suggetion.

Renovate supports slash-separated tags now (https://github.com/renovatebot/renovate/pull/35431) and the action is actually pinned to a hash.